### PR TITLE
2 bugs fixing

### PR DESCRIPTION
**Pull request by sintinilorenzo**

### CODE REVIEW

File: src/components/organisms/SchedulePageModal/SchedulePageModal.tsx [hunk 1]
```diff
@@ -106,6 +106,8 @@ export const SchedulePageModal: React.FunctionComponent<PropsType> = ({
   }, [openVenues]);
 
   const [date, setDate] = useState(0);
+  console.log(date);
+  console.log(orderedEvents);
```
Comment (maxb94) [inline src/components/organisms/SchedulePageModal/SchedulePageModal.tsx:110]: .

File: src/components/molecules/SparkleFairiesPopUp/SparkleFairiesPopUp.tsx [hunk 2]
```diff
@@ -11,21 +18,38 @@ const SparkleFairiesPopUp = () => {
...
+          <button
+            onClick={() => setShowEventSchedule(true)}
+            className="btn btn-primary join-button button-event-schedule"
+            style={{ fontSize: 14 }}
+          >
+            Events Schedule
+          </button>
+          <br />
+          <a
+            href="https://playa.sparklever.se/in/playainfo"
```
Comment (maxb94) [inline src/components/molecules/SparkleFairiesPopUp/SparkleFairiesPopUp.tsx:33]: add to settings.ts

File: src/components/molecules/SparkleFairiesPopUp/SparkleFairiesPopUp.tsx [hunk 3]
```diff
@@ -11,21 +18,38 @@ const SparkleFairiesPopUp = () => {
...
+            target="_blank"
+            className="btn btn-primary join-button"
+            style={{ fontSize: 14 }}
+          >
+            Playa Information Booth
+          </a>
+          <div className="title">Help and support</div>
+          <p className="title-sidebar">{`If you're in need of help or support, you can call on our Reality Rangers`}</p>
+          <a
+            href="https://multiverserangers.org/rangers911/"
```
Comment (maxb94) [inline src/components/molecules/SparkleFairiesPopUp/SparkleFairiesPopUp.tsx:44]: add to settings.ts

**Review by maxb94 — APPROVED** [review]

**Review by thecadams — COMMENTED** [review]
Just needs the fixes suggested by Max and all good

**Pull request merged**